src/components/prescriptions/PrescritionImport.vue

```diff
@@ -0,0 +1,257 @@
+
+    const nameAndSurnameDoctor = ref(null);
+    const medicine = ref(null);
+    const frequency = ref(null);
+    const researchDate = ref(null);
+    const successFlag = ref(false);
+    const testFile = ref(null);
+    const testFileError = ref(null);
+
+    const userStore = useUserStore();
+
+    const disableAddTest = computed(() => !testFile.value);
+    const dateFormat = computed(
+        () =>
+            researchDate.value &&
+            researchDate.value.split('-').reverse().join('-')
+    );
+
+    const handleChange = (event) => {
+        const selectedFile = event.target.files[0];
+        if (selectedFile && ALLOWED_TYPES.includes(selectedFile.type)) {
+            testFile.value = selectedFile;
+            testFileError.value = null;
+        } else {
+            testFile.value = null;
+            testFileError.value =
+                'Please select an image file (jpeg, jpg, png, webp or pdf).';
+        }
+    };
+
+    const addTest = async () => {
+        try {
+            await uploadImage(testFile.value);
+            console.log(`image uploaded`, url.value);
+            await addDoc({
+                userId: userStore.getUserId,
+                dateImport: dateFormat.value,
+                doctorDataImport: nameAndSurnameDoctor.value,
+                medicineNameImport: medicine.value,
+                frequencyMedicineImport: frequency.value,
+                testUrl: url.value,
+                createdAt: timestamp(),
+            });
+            if (error.value) throw new Error();
+            isLoading.value = false;
+            successFlag.value = true;
+        } catch (err) {
+            console.log(err);
+            console.log(error.value);
+            console.log(error.message);
+        } finally {
+            nameAndSurnameDoctor.value = null;
+            medicine.value = null;
+            frequency.value = null;
+            testFile.value = null;
+            testFileError.value = null;
+            researchDate.value = null;
+        }
+    };
+</script>
+
+<style lang="scss" scoped>
+    .prescriptionBox {
+        @include flex-position(column, nowrap, space-between, flex-start);
+        height: 100%;
+        &__header {
+            @include text-header3($font-weight-semiBold);
+            color: $blue-900;
+        }
+        &__form {
+            height: 100%;
+            select,
+            input {
+                @include text-header6($font-weight-regular);
+            }
+            label {
+                @include label;
+                color: $blue-900;
+            }
+            .actionsAndInfo {
+                @include flex-position(column, nowrap, center, center);
+                gap: 1rem;
+                align-self: center;
+            }
+        }
+        .prescriptionForm {
+            width: 100%;
+            @include flex-position(column, nowrap, space-evenly, flex-start);
+            gap: 1rem;
+            &__bloodSection {
```

Comment:
Pls dont copy. This selector not work. If u want you have to change class name and selector. Otherwise we treat this as rubbish and should be deleted.